cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -188,8 +213,91 @@ instance DefineSeverity (ChainDB.TraceEvent blk) where
             _ -> Debug
     defineSeverity (ChainDB.TraceImmDBEvent _ev) = Debug
 
+instance DefinePrivacyAnnotation (TraceChainSyncClientEvent blk)
+instance DefineSeverity (TraceChainSyncClientEvent blk) where
+    defineSeverity (TraceDownloadedHeader _) = Info
```

Comment:
Indent should be two spaces. We have a style document here about to be merged that should be adhered to: https://github.com/input-output-hk/cardano-node/pull/164

Reply:
negative! I am not going to change the indentation just because there has a somewhat "new" style guide emerged without consent.

Reply:
@CodiePP Then please submit your feedback on https://github.com/input-output-hk/cardano-node/pull/164 :).